src/security/pages/sign-up.component.vue

```diff
@@ -0,0 +1,59 @@
+<script setup>
+import InputText from "primevue/inputtext";
+import ToggleButton from "primevue/togglebutton";
+import { ref } from "vue";
+
+const checked1 = ref(false);
+</script>
+
+<template>
+  <div class="flex justify-center items-center">
+    <div class="flex flex-col container max-w-md items-center mt-8">
+      <div class="py-6">
```

Comment:
Padding is mostly used to define space between the border of the element and the content, not to define space between this element and another one. You should change this to use a margin instead:

```suggestion
      <div class="mb-6">
```